### DISCUSSION TITLE
The Guidelines of Discussions

### DISCUSSION THREAD

**Discussion by gujun720** [milvus-io/milvus#3420] [Announcement]
Hi, there

Welcome to join the discussions with the Milvus community. Before starting a new discussion, please follow the below guidelines.

1. English is the only communication language here, as it is an incubation project at the LF AI & DATA foundation.
2. If you have a question, please first try to search the document and the existing discussions.

Wish you enjoy the interaction with the Milvus community.

**Comment by walker1507** [comment]
Yes I am really interested 
But how do we start anything's

**Reply by xiaofan-luan** [reply]
> Yes I am really interested But how do we start anything's

Hi walker, did you have any background on vector search, distributed system, SQL, restful or any language?
You can work with the sdk maintainer for the SDK contribution

**Comment by LMERose** [comment]
Hello, I would like to ask if milvus provides an LRU（Least Recently Used） method? How to use it?

**Comment by lontion** [comment]
I want to use the sift data set to test different search methods in milvus, such as "index_type": "IVF_FLAT", "metric_type": "L2". But the recall rate achieved so far is always 0. Is there anyone who has done this before? Can you help me answer it?

**Reply by yhmo** [reply]
Kindly remember that this discussion is not the right place to post questions. You should click the "New Discussion" button to create a separate discussion for your question.

**Comment by Shagga01** [comment]
Hi, I am Humphrey O. Founder and Team Leader of Red Waters Group. 
Red Waters Group is assembling a high-caliber global team of innovators, builders, and digital strategists.
We’re creating a future-forward ecosystem where AI, cloud engineering, automation, cybersecurity, product innovation, and digital architecture converge. If you’re a skilled professional ready to build revolutionary systems, contribute your expertise, and grow with a team driven by ambition and impact—we want you with us. Join us as we engineer the next generation of digital solutions.
👉 Apply here: https://forms.gle/RgLpGKKn5MZMVbRq9
